Bottom: 74f3472b1382bbf2fc254616c79de66d78e597d5
Top:    d8cf9acc32cd9a99020ff475e5e60675266d2ea2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-21 13:08:42 +0000

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!









---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index c36d23c..c1c4e89 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -332,13 +332,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
